src/bitcoincash_validator.ts

```diff
@@ -0,0 +1,142 @@
+import BCH from "bchaddrjs-slp";
+
+import { NetTypes } from "./types/net.types";
+import {
+  AddressFormats,
+  TAddress,
+  TBaseValidator,
+  TIsValidAddress,
+  TNetType,
+} from "./types/validators.types";
+
+const DEFAULT_ADDRESS_FORMAT = AddressFormats.legacy;
+const DEFAULT_NETWORK_TYPE = NetTypes.prod;
+
+const isAValidAddress: TIsValidAddress = (address) => {
+  try {
+    BCH.decodeAddress(address);
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+// TODO refactor
```

Comment:
Is it done/resolved?

Reply:
Same reason as [here](https://github.com/RampNetwork/crypto-address-validator/pull/1#discussion_r804724328)